services/distribution/src/main/java/app/coronawarn/server/services/distribution/assembly/appconfig/ApplicationConfigurationProvider.java

```diff
@@ -30,30 +30,13 @@
  *   <li>Risk Score Classification</li>
  *   <li>App Config, e.g. minimum risk threshold</li>
  * </ul>
- *
- * <p>The application config is fetched from the master-config folder.</p>
  */
 public class ApplicationConfigurationProvider {
 
-  /**
-   * The location of the exposure configuration master file.
-   */
-  public static final String MASTER_FILE = "master-config/app-config.yaml";
```

Comment:
Not sure about this change tbh - I don't see any reason why somebody would want to have that file path configurable at all.